client/Components/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import { Button } from '@material-ui/core';
+
+const Button = () => {
+  return <Button style={{ color: 'white' }}>Log in</Button>;
```

Comment:
im not a fan of inline styles- i would prefer to use either 
withStyles or useStyles
```js
<Button className={classes.whateverClassName}>sklajdklsajd</Button>
```